draft-ietf-httpbis-message-signatures.md

```diff
@@ -300,11 +300,11 @@ The following example shows canonicalized values for different component identif
 "x-dictionary";key="c": (a, b, c)
 ~~~
 
-## Specialty Components {#specialty-content}
+## Derived Components {#derived-components}
 
-Message components not found in an HTTP field can be included in the signature input by defining a component identifier and the canonicalization method for its component value.
+In addition to HTTP fields, there are a number of different components that can be derived from the control data, processing context, or other aspects of the HTTP message being signed. Such derived components can be included in the signature input by defining a component identifier and the derivation method for its component value.
 
-To differentiate specialty component identifiers from HTTP fields, specialty component identifiers MUST start with the "at" `@` character. This specification defines the following specialty component identifiers:
+To differentiate for identifiers derived components from identifiers for HTTP fields, derived component identifiers MUST start with the "at" `@` character. This specification defines the following derived component identifiers:
```

Comment:
I find this sentence quite hard to grok. Are you trying to say something like

Derived component identifiers MUST start with the "at '@' character. This avoids derived component identifiers being mistake for HTTP fields.

Reply:
Yes, something like that. I'll reword it, thanks.